rollup/rollupConfigBuilder.js

```diff
@@ -0,0 +1,39 @@
+const ui5 = require("./ui5-resolve");
+const builtins = require('builtin-modules');
+const { terser } = require("rollup-plugin-terser");
+const { nodeResolve } = require("@rollup/plugin-node-resolve");
+
+class ConfigBuilder {
+    static build(projectPath) {
+        return {
+            input: "./rollup/bundle-def.js",
+            plugins: [
+                ui5({
+                    assets: [
+                        "/resources/sap/ui/fl/**"
+                    ],
+                    exclude: [
+                        "./rollup/bundle-def.js",
+                        "sap/ui/thirdparty/URI"
```

Comment:
why do you have the overlay here and exclude it?

Reply:
https://github.com/SAP/openui5/blob/master/src/sap.ui.core/src/sap/ui/thirdparty/URI.js was particulary difficult to convert. Either it required modification of the module, or override. Maybe the "exclude" word is misleading. It should be something like "skipTransformation".

Reply:
👍